tests/compile-fail/overflow_check_conditional.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#![deny(overflow_check_conditional)]
+
+fn main() {
+	let a: u32 = 1;
+	let b: u32 = 2;
+	if a + b < a {
+
+	}
```

Comment:
We you write a test, add things that are similar but should not lint, eg. `a + b < c`. Also `a + b < c` should lint as well.